src/components/LibraryItem.tsx

```diff
@@ -158,8 +158,9 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     getIconElement(): JSX.Element {
-        // Group displays only text without icon.
-        if (this.props.data.itemType !== "group" && this.props.data.itemType !== "section") {
+        let itemType = this.props.data.itemType;
+        // If the element type is a section, group, or none an icon shouldn't be displayed
+        if (itemType !== "group" && itemType !== "section" && itemType !== "none") {
```

Comment:
may be you should check the tests and see this is covered.

Reply:
@ramramps yes will do, the AC calls for a unit test